_posts/ja/posts/2018-09-18-release-0.16.3.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Bitcoin Core 0.16.3 リリース
+name: blog-release-0.16.3
+id: ja-blog-release-0.16.3
+lang: ja
+type: posts
+layout: post
+share: true
```

Comment:
Can you please add the following line so that this has the same URL path as previous translated release announcements?

    permalink: /ja/2018/09/18/release-0.16.3/

Reply:
oh, my mistake. added.